--- library/mysql-resultset/source/MYXResultSetStorage.cc
+++ library/mysql-resultset/source/MYXResultSetStorage.cc
@@ -423,9 +423,9 @@
 
 
 
+/*
 MYXRSCompareStatus MYXResultSetStorage::get_compare_status(bigint row, int column)
 {
-  /*
   switch (_resultset->rows[row].diff&MYX_RD_MASK)
   {
     case MYX_RD_OTHER_ONLY:
@@ -438,8 +438,9 @@
         return MCSDiffers;
     default:
       return MCSMatch;
-  }*/
+  }
 }
+*/
 
 
 MYXRSEditStatus MYXResultSetStorage::get_edit_status(bigint row, int column) const
